modules/cluster-logging-rn-5.5.4.adoc

```diff
@@ -0,0 +1,45 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-4_{context}"]
+= Logging 5.5.4
+This release includes link:https://access.redhat.com/errata/RHSA-2022:7434[RHSA-2022:7434-OpenShift Logging Bug Fix Release 5.5.4].
+
+[id="openshift-logging-5-5-4-bug-fixes"]
+== Bug fixes
+* Before this update, an error in the query parser of the logging view plugin caused parts of the logs query to disappear, if the query contained curly brackets `{}`. This made the queries invalid, leading to errors being returned for valid queries. With this update, the parser correctly handles these queries. (link:https://issues.redhat.com/browse/LOG-3042[LOG-3042])
+
+* Before this update, the Operator could enter a loop of removing and recreating the collector daemonset while the Elasticsearch or Kibana deployments changed their status. With this update, a fix in the status handling of the Operator resolves the issue. (link:https://issues.redhat.com/browse/LOG-3049[LOG-3049])
+
+* Before this update, there were no alerts implemented to support the collector implementation of Vector. This change adds Vector alerts and deploys separate alerts depending upon the chosen collector implementation. (link:https://issues.redhat.com/browse/LOG-3127[LOG-3127])
+
+* Before this update, the secret creation component of the Elasticsearch Operator modified internal secrets constantly. With this update, the existing secret is properly handled. (link:https://issues.redhat.com/browse/LOG-3138[LOG-3138])
+
+* Before this update, a prior refactoring of the logging `must-gather` scripts removed the expected location for the artifacts. This update reverts that change to write artifacts to the `/must-gather` folder. (link:https://issues.redhat.com/browse/LOG-3213[LOG-3213])
+
+* Before this update, on certain clusters, the Prometheus exporter would bind on IPv4 instead of IPv6. After this update, Fluentd detects the IP version (v4 or v6), and the binds to `0.0.0.0` for IPv4 and binds to `[::]` for IPv6. (link:https://issues.redhat.com/browse/LOG-3162[LOG-3162])
```

Comment:
```suggestion
* Before this update, on certain clusters, the Prometheus exporter would bind on IPv4 instead of IPv6. After this update, Fluentd detects the IP version (v4 or v6) and binds to `0.0.0.0` for IPv4 or `[::]` for IPv6. (link:https://issues.redhat.com/browse/LOG-3162[LOG-3162])
```